.eslintrc.js

```diff
@@ -0,0 +1,68 @@
+module.exports = {
```

Comment:
Once moved, this will extend from the root eslintrc in @sentry-javascript. The only thing we'll need to keep is a couple of ignore patterns and rules